src/Microsoft.DotNet.Darc/src/DarcLib/Helpers/GitFileManager.cs

```diff
@@ -156,10 +177,12 @@ public async Task<IEnumerable<DependencyDetail>> ParseVersionDetailsXmlAsync(str
                 }
 
                 XmlNode nodeToUpdate = versionList.Item(0);
-                nodeToUpdate.Attributes["Version"].Value = itemToUpdate.Version;
-                nodeToUpdate.Attributes["Name"].Value = itemToUpdate.Name;
-                nodeToUpdate.SelectSingleNode("Sha").InnerText = itemToUpdate.Commit;
-                nodeToUpdate.SelectSingleNode("Uri").InnerText = itemToUpdate.RepoUri;
+
+                SetAttribute(versionDetails, nodeToUpdate, "Version", itemToUpdate.Version);
+                SetAttribute(versionDetails, nodeToUpdate, "Name", itemToUpdate.Name);
+                SetAttribute(versionDetails, nodeToUpdate, "SourceBuildId", itemToUpdate.BuildId.ToString());
```

Comment:
>SourceBuildId [](start = 60, length = 13)

`DependencyDetail.cs` has `BuildId` and `SourceBuildId` is used here, I'd just map them both